datalad_tabby/io/__init__.py

```diff
@@ -40,7 +41,10 @@ def load_tabby(
 
     Other tabby tables/sheets are loaded when ``@tabby-single|many-`` import
     statements are discovered. The corresponding data structures then replace
-    the import statement at its location.
+    the import statement at its location. Recursive loading of other tables
+    can be deactivated by setting the ``mode`` parameter to ``nonrecursive``,
+    which will result in only the record available at the ``src`` path being
+    loaded.
```

Comment:
```suggestion
    the import statement at its location. Setting the ``recursive`` flag to ``False``
    disables table import, which will result in only the record available at the
    ``src`` path being loaded.
```

The is no ``mode`` parameter for this function.